repose-aggregator/components/filters/valkyrie-authorization-filter/build.gradle

```diff
@@ -20,7 +22,10 @@ dependencies {
     testCompile project(":repose-aggregator:commons:commons-test")
     testCompile "junit:junit"
     testCompile "org.scalatest:scalatest_$scalaMajDotMin"
-    testCompile "org.mockito:mockito-all"
+    testCompile "org.hamcrest:hamcrest-all"
+    testCompile("org.mockito:mockito-all") {
```

Comment:
Probably don't want all if we are to exclude hamcrest

Reply:
Fair enough. I don't think I checked at the time, but the `mockito-all` artifact just combines `mockito-core` with an `org.hamcrest` package and an `org.objenesis` package. Since we aren't using `objenesis` ourselves, I don't think we need it, so I'll try to downgrade this to `mockito-core`.

As a side note, `hamcrest` is clearly not packaged within `mockito-core`. I wonder why I added an exclusion for it even where I switched to `mockito-core` dependencies. I guess it might be a transitive. I'll verify that the exclusion is necessary and remove it if not.

Reply:
I'd move away from using all everywhere if we can